spec/features/consumer/shopping/unit_price_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+
+feature "As a consumer, I want to check unit price information for a product ;", js: true do
+  include AuthenticationHelper
+  include WebHelper
+  include ShopWorkflow
+  include UIComponentHelper
+
+  # Create context
+  let(:distributor) { create(:distributor_enterprise, with_payment_and_shipping: true) }
+  let(:supplier) { create(:supplier_enterprise) }
+  let!(:order_cycle) { create(:simple_order_cycle, distributors: [distributor], variants: [product.variants.first]) }
+  let(:product) { create(:simple_product, supplier: supplier) }
+  let(:variant) { product.variants.first }
+  let(:order) { create(:order, order_cycle: order_cycle, distributor: distributor) }
+  let(:exchange1) { order_cycle.exchanges.to_enterprises(distributor).outgoing.first }
+  let(:user) { create(:user, password: "password", password_confirmation: "password") }
+
+  variantUnitPrice = '.variant-unit-price'
+  questionMarkIcon = '.question-mark-icon'
+  tooltip = '.joyride-tip-guide.question-mark-tooltip'
+
+  before do
+    set_order order
+    exchange1.update_attribute :pickup_time, "monday"
+    add_variant_to_order_cycle(exchange1, variant)
+    allow(OpenFoodNetwork::FeatureToggle)
+      .to receive(:enabled?).with(:unit_price, anything) { true }
+  end
+
+  describe "for the shopfront" do
+    before do
+      visit shop_path
+    end
+    
+    it "it shows the unit price information with the question mark icon." do
+      expect(page).to have_selector variantUnitPrice
+      within variantUnitPrice do 
+        expect(page).to have_selector questionMarkIcon
+      end
+    end
```

Comment:
given we're not using the browser at all and just checking the rendered page, I'd rather move this to a view test under `spec/views`. It feels to me like a unit-test and not an end2end one. That would also be dramatically faster.

Reply:
There you go: 7651f97b90b066007a63defb2f880ac92a6f8fe2

Reply:
Any thoughts?

Reply:
that looks good :+1: